### PR TITLE
Polished user experience

### PR DESCRIPTION
**Pull request by djyou**
Changed containerregistry to cr and removed -g option from show/update/delete commands.

### CODE REVIEW

File: lib/plugins.arm.json [hunk 1]
```diff
@@ -34761,8 +34734,8 @@
         {
           "name": "search",
           "description": "Search for a repository name under a container registry",
-          "fullName": "containerregistry search",
-          "usage": "[options] registry",
+          "fullName": "cr search",
```
Comment (SajayAntony) [inline lib/plugins.arm.json]: Not as a part of this commit but, we might have to consider removing search.

**Comment by SajayAntony** [comment]
:shipit:

**Comment by DavidObando** [comment]
:shipit:

**Pull request merged**